test/unit/tool_waitlist_test.rb

```diff
@@ -25,7 +25,62 @@
 require 'test_helper'
 
 class ToolWaitlistTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+#Relationships 
+  should belong_to(:tool_type)
+  should belong_to(:organization)
+  should belong_to(:participant)
+
+  #Validations 
+  should validate_presence_of(:tool_type)
+  should validate_presence_of(:organization)
+  should validate_presence_of(:participant)
+  should validate_presence_of(:wait_start_time)
+
+  context "With a proper context, " do
+    setup do
+    # creating people for waitlist 
+   # @person1 = FactoryGirl.create(:participant, :andrewid => 'ersmith')
+    @person2 = FactoryGirl.create(:participant, :andrewid => 'erbob')
+    @person3 = FactoryGirl.create(:participant, :andrewid => 'ersam')
+
+    #making tool types 
+     @saw = FactoryGirl.create(:tool_type, name: 'Saw')
+     @hammer = FactoryGirl.create(:tool_type, name: 'Hammer')
```

Comment:
Thanks for point it out! I did not feel like it was a huge problem so I merged the PR. But we will try to avoid this kinds of problem next time